appveyor.yml

```diff
@@ -116,7 +112,7 @@ after_test:
       }
       .\cache-s3 --prefix=$env:APPVEYOR_PROJECT_NAME --git-branch=$env:APPVEYOR_REPO_BRANCH --suffix=windows -c -v info save stack work
     }
- - xcopy /q /s /e /r /k /i /v /h /y "%WORK_DIR%\daedalus" "%APPVEYOR_BUILD_FOLDER%\daedalus"
+# - xcopy /q /s /e /r /k /i /v /h /y "%WORK_DIR%\daedalus" "%APPVEYOR_BUILD_FOLDER%\daedalus"
```

Comment:
Afaik this was needed for the artifacts so it should stay. If not, we should just remove it but not comment it out.

Reply:
Of course it's needed 🤦‍♂️ . Let me take care of that.